src/main/java/cms/gov/madie/measure/resources/ExportController.java

```diff
@@ -82,4 +82,32 @@ public ResponseEntity<byte[]> getTestCaseExport(
     return fhirServicesClient.getTestCaseExports(
         measure, accessToken, testCaseId, bundleType.orElse(("COLLECTION").toUpperCase()));
   }
+
+  @GetMapping(path = "/measures/{id}/qdm/qrda", produces = "application/zip")
```

Comment:
You have set a path to export both QDM and QICore. makes more sense to not make the endpoint QDM specific although I do not know if QRDA is used for QICore test cases.
```suggestion
  @GetMapping(path = "/measures/{id}/test-cases/qrda", produces = "application/zip")
```

Reply:
@adongare This is changed.